docs/deployment/FPC_on_Azure.md

```diff
@@ -0,0 +1,139 @@
+# How to use an Azure Confidential Computing Instance to run FPC in HW mode
+Date of writing 14.03.2022.
+This guide is based on the following sources.
+[Koshi Ikegawa](https://qiita-com.translate.goog/ikegawa-koshi/items/8cf1fef1004fc16beb15?_x_tr_sl=ja&_x_tr_tl=en&_x_tr_hl=de&_x_tr_pto=wapp#fpc%E3%81%AE%E3%83%93%E3%83%AB%E3%83%89)
+
+- [How to use an Azure Confidential Computing Instance to run FPC in HW mode](#how-to-use-an-azure-confidential-computing-instance-to-run-fpc-in-hw-mode)
+  - [Creating the Confidential Computing Instance on Azure.](#creating-the-confidential-computing-instance-on-azure)
+  - [Prerequisites](#prerequisites)
+  - [Registering for the SGX Attestation Service Utilizing EPID](#registering-for-the-sgx-attestation-service-utilizing-epid)
+    - [Known Issues](#known-issues)
+  - [Building the FPC dev container](#building-the-fpc-dev-container)
+  - [Final preparation of the dev-container](#final-preparation-of-the-dev-container)
+
+## Creating the Confidential Computing Instance on Azure.
+Use the [Quick Create Portal](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/quick-create-portal) to create
+the virtual machine.
+Use the following parameters:
+* OS = *Linux (ubuntu 20.04)*
+* Size = Standard DC2s v2 (2 vcpus, 8 GiB memory)
+
+## Prerequisites
+Connect to your instance using ssh and execute the following commands
+```bash
+#update machine
+sudo -i
+apt-get update
+apt-get upgrade
+apt install -y docker docker-compose golang make
+exit
+#install sgx environment
+#1.add sgx repo + key and then install packages form this repository
+echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
+wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo apt-key add -
+sudo apt-get update
+sudo apt -y install libssl-dev libsgx-enclave-common libsgx-enclave-common-dev libsgx-ae-qe3 libsgx-ae-qve libsgx-epid libsgx-launch libsgx-pce-logic libsgx-qe3-logic libsgx-quote-ex libsgx-uae-service libsgx-urts
+sudo reboot
+```
+After reboot is completed connect to your vm again.
+```bash
+#check if asemd service is running
+sudo systemctl status aesmd.service
+```
+```bash
+sudo usermod -aG docker $(whoami)
+go env -w GO111MODULE=auto
+
+echo 'export GOPATH=$HOME/go' >> .bashrc
+echo 'export GOBIN=$GOPATH/bin' >> .bashrc
+echo 'export PATH=$PATH:$GOBIN' >> .bashrc
+
+sudo reboot
+``` 
+After reboot is completed connect to your vm again.
+```bash
+mkdir -p $GOPATH
+cd $GOPATH
+go get github.com/hyperledger/fabric-private-chaincode
+cd  ~
```

Comment:
oh! I would not recommend getting FPC in this way. Better like this.
```
export FPC_PATH=$GOPATH/src/github.com/hyperledger/fabric-private-chaincode
git clone --recursive https://github.com/hyperledger/fabric-private-chaincode.git $FPC_PATH
```

Reply:
Could you elaborate why getting fpc with go get is unadvisable?
Is it because --recursive is not possible with go get and therefore it would fail to clone the missing sub-repositories?

Reply:
will try wit "normal" git instead of go get